backend/mj/spiders/__init__.py

```diff
@@ -11,6 +12,17 @@ def process_item(self, item, spider):
         return item
 
 
+class SanitizeHTMLPipeline:
+
```

Comment:
```suggestion
    """For security, escapes or removes all html tags from chosen fields, e.g. event description."""
    
```

does this docstring describe what's happening?

Reply:
Yep, added via fb27218